docs/products/clickhouse/howto/configure-tiered-storage.rst

```diff
@@ -0,0 +1,94 @@
+Configure data retention thresholds in Aiven for ClickHouse®'s tiered storage
+=============================================================================
+
+.. important::
+
+    Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Learn to control how your data is distributed between storage devices in the tiered storage of an Aiven for ClickHouse service. Check out how to configure tables so that your data is automatically written either to SSD or the object storage as needed.
+
+About data retention control
+----------------------------
+
+If you have the tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on your Aiven for ClickHouse service, your data is distributed between two storage devices (tiers). The data is stored either on SSD or in the object storage, depending on whether and how you configure this behavior. If you don't, by default the data is moved from SSD to the object storage when SSD reaches 80% of its capacity (default size-based data retention policy).
+
+You may want to change this default data distribution behavior by :ref:`configuring your table's schema by adding a TTL (time-to-live) clause <time-based-retention-config>`. Such a configuration allows ignoring the SSD-capacity threshold and moving the data from SSD to the object storage based on how long the data is there on your SSD.
+
+To enable this time-based data distribution mechanism, you can set up a retention policy (threshold) on a table level by using the TTL clause. For data retention control purposes, the TTL clause uses the following:
+
+* Data item of the `Date` or `DateTime` type as a reference point in time
```

Comment:
Maybe 'A data item' ?
And 'An INTERVAL clause'
Just sounds a bit nicer to me with the article, but it's not incorrect as it is.

Reply:
this is actually a good practice in tech writing to omit articles at the beginning of the bullet list items so I'd keep it as it is if you don't mind, at least for consistency.